doc/WebSite/Dapper-Usage.html

```diff
@@ -0,0 +1,187 @@
+			Dapper is free as open source software that is distributed under dual license, either the Apache License 2.0 or the MIT License.
+			<a href="https://www.nuget.org/packages/Abp.Dapper" target="_blank">Abp.Dapper</a> package simply integrates Dapper to ASP.NET Boilerplate.</p>
+
+			<h3 id="DocQuickStart">Quick Start</h3>
+			<p>Before you start, you need to download <a href="https://www.nuget.org/packages/Abp.Dapper" target="_blank">Abp.Dapper</a> and <a href="https://www.nuget.org/packages/Abp.EntityFrameworkCore" target="_blank">Abp.EntityFrameworkCore</a> nuget packages 
+			to project that you want to use it.
+			</p>
+			
+			<h4>Module Registration</h3>
+			
+			<p>First you need to add <strong>DependsOn</strong> attribute for <strong>AbpDapperModule</strong> on your module where you register it.</p>
+			
+			<p><strong>NOTE:</strong> Modules need to be added by order. And also you should define mappings, if your entities and tables don't have the same name.</p>
+			
+			<pre lang="cs">[DependsOn(
+     typeof(AbpEntityFrameworkModule),
+     typeof(AbpDapperModule)
+)]
+public class SampleApplicationModule : AbpModule
+{
+    public override void Initialize()
+    {
+        IocManager.RegisterAssemblyByConvention(typeof(SampleApplicationModule).GetAssembly());
+    }
+}
+			</pre>
+			
+			<h4>When entity classes and tables have different names</h4>
+
+			<p>For example, <strong>User</strong> class reference <strong>AbpUsers</strong> table. So you should define a mapping between them.</p>
+			
+			<pre lang="cs">public class UserMapper : ClassMapper&lt;User&gt;
+{
+    public UserMapper()
+    {
+        //use different table name
+        Table("AbpUsers");
+        
+        Map(x => x.Logins).Ignore();
+        Map(x => x.Roles).Ignore();
+        Map(x => x.Claims).Ignore();
+        Map(x => x.Permissions).Ignore();
+        Map(x => x.Settings).Ignore();
+
+        Map(x => x.DeleterUser).Ignore();
+        Map(x => x.CreatorUser).Ignore();
+        Map(x => x.LastModifierUser).Ignore();
+        Map(x => x.FullName).Ignore();
+        Map(x => x.Tokens).Ignore();
+
+        AutoMap();
+    }
+}</pre>
+			
+			<p>After define mappings, you need to register them in module initialize method.</p>
+						<pre lang="cs">[DependsOn(
+     typeof(AbpEntityFrameworkModule),
+     typeof(AbpDapperModule)
+)]
+public class SampleApplicationModule : AbpModule
+{
+    public override void Initialize()
+    {
+        IocManager.RegisterAssemblyByConvention(typeof(SampleApplicationModule).GetAssembly());
+        <strong>DapperExtensions.DapperExtensions.SetMappingAssemblies(new List<Assembly> { typeof(SampleApplicationModule).GetAssembly() });</strong>
+    }
+}
+			</pre>
+			
+			<h3 id="DocUsage">Usage</h3>
+
+			
+			<p>After registering <strong>AbpDapperModule</strong>, you can use it like traditinal ABP repositories like following.</p>
```

Comment:
"you can use it like traditinal ABP repositories" is wrong because traditional ABP repositories are used with IRepository, but this one used with IDapperRepository. This could be better:

"You can use Generic IDapperRepository interface (instead of standard IRepository) to inject dapper repositories."